src/reader_lcf.cpp

```diff
@@ -7,6 +7,7 @@
  * file that was distributed with this source code.
  */
 
+#include <stdio.h>
```

Comment:
This file already has this include for the NDEBUG case a few lines below. Please remove the 3 lines of NDEBUG check below, too.